packages/aws-cdk/lib/version.ts

```diff
@@ -52,15 +72,19 @@ export class TimestampFile {
     }
   }
 
-  public async update(): Promise<void> {
+  public async update(latestVersion?: string): Promise<void> {
     const fd = await open(this.file, 'w');
```

Comment:
There is also `fs.writeFile`, seems simpler: https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options

Reply:
Definitely simpler. Fixed it up.